show/interfaces/__init__.py

```diff
@@ -319,6 +319,52 @@ def expected(db, interfacename):
 
     click.echo(tabulate(body, header))
 
+# 'mpls' subcommand ("show interfaces mpls")
+@interfaces.command()
+@click.argument('interfacename', required=False)
+@multi_asic_util.multi_asic_click_options
```

Comment:
for now remove the 'multi_asic_click_options' and the 'namespace' argument. Support for multi ASIC platform is requried but it would be hard to test and write unit tests without multi ASIC platform. (It is possible to test with multi ASIC VM)